docs/nightly/en/getting-started/quick-start/prerequisites.md

```diff
@@ -12,16 +12,16 @@ services:
     image: grafana/grafana-oss:<%grafana-version%>
     container_name: grafana
     ports:
-      - 3000:3000
+      - 127.0.0.1:3000:3000
```

Comment:
Why is `127.0.0.1:` necessary?

Reply:
https://docs.docker.com/network/#published-ports